app/Http/Resources/DoctorResource.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class DoctorResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'type' => 'doctors',
+            'id' => $this->id,
+            'attributes' => [
+                'agenda' => $this->agenda,
+                'created_at' => $this->created_at->toDateTimeString(),
+                'name' => $this->name,
+                'updated_at' => $this->updated_at->toDateTimeString(),
+            ],
+            'relationships' => [
+                'availabilities' => [
+                    'data' => $this->availabilities()->exists() ? $this->availabilities->map(function ($availability) {
+                        return [
+                            'type' => 'availabilities',
+                            'id' => $availability->id
+                        ];
+                    }) : null,
```

Comment:
In an API we don't always want the related resources as there can be a lot of them. Instead you can use [whenLoaded](https://laravel.com/docs/8.x/eloquent-resources#conditional-relationships) to send the related resources only when necessary.

```suggestion
                'availabilities' => [
                    'data' => $this->whenLoaded('availabilities'),
```